src/dd-draggable.ts

```diff
@@ -184,6 +178,12 @@ export class DDDraggable extends DDBaseImplement implements HTMLElementExtendOpt
     // console.log(`${count++} move ${e.x},${e.y}`)
     let s = this.mouseDownEvent;
 
+    let node = (this.el as GridItemHTMLElement)?.gridstackNode;
```

Comment:
no, you wan tot move this to 203

this.dragging = true;
DDManager.dragElement = this;
... then your code